src/css/base.scss

```diff
@@ -219,20 +215,51 @@ footer {
     background: white;
   }
 }
-@media (max-width:550px) {
+@media (max-width:700px) {
```

Comment:
I'm seeing some issues with this break point and the login in getting a little funky as the screen gets smaller. let's take a look at this together